standards-catalogue/docs/standard_template.md

```diff
@@ -0,0 +1,14 @@
+```
+---
```

Comment:
I'm thinking that this should go into i.e. `content/standards/_template.md`, as then it'll get auto-validated, thoughts?

Reply:
it could but it would fail most of the validation tests because of the empty values

Reply:
could put some dummy values in there - but then we might as well not have a template and point them to an existing standard to copy and paste, and just change the values

Reply:
Mmm, that's fair. "Copy from an existing one" makes sense, as it actually has some real-world data they can copy :+1: